org.mixedrealitytoolkit.uxcore/Slider/Slider.cs

```diff
@@ -153,9 +153,16 @@ public float Value
             {
                 if (this.value != value)
                 {
-                    var oldSliderValue = this.value;
-                    this.value = value;
-                    OnValueUpdated.Invoke(new SliderEventData(oldSliderValue, value));
+                    if (value < minValue || value > maxValue)
+                    {
+                        Debug.Log($"{value} is outside of the slider range.");
```

Comment:
I think we should instead clamp the value. 

`value = Mathf.Clamp(value, minValue, maxValue)`

Also, I'm not sure how I feel about this log message, maybe just remove?

Reply:
will change